apps/server/src/modules/task-card/controller/api-test/task-card.api.spec.ts

```diff
@@ -411,11 +363,71 @@ describe('Task-Card Controller (api)', () => {
 					.expect(200);
 
 				const responseTaskCard = response.body as TaskCardResponse;
-				const richTextElement = responseTaskCard.cardElements.filter(
+				const richTextElement = responseTaskCard.cardElements?.filter(
 					(element) => element.cardElementType === CardElementType.RichText
 				);
-				expect((richTextElement[0].content as CardRichTextElementResponse).value).toEqual(sanitizedText);
+				expect(richTextElement ? richTextElement[0].content.value : '').toEqual(sanitizedText);
 			});
 		});
 	});
+	describe('When title is provided', () => {
+		it('should create a task card with title', async () => {
+			const user = setupUser([Permission.TASK_CARD_EDIT, Permission.HOMEWORK_CREATE, Permission.HOMEWORK_EDIT]);
+
+			await em.persistAndFlush([user]);
+			em.clear();
+
+			currentUser = mapUserToCurrentUser(user);
+
+			const taskCardParams = {
+				title: 'test title',
+			};
+
+			const response = await request(app.getHttpServer())
+				.post(`/cards/task/`)
+				.set('Accept', 'application/json')
+				.send(taskCardParams)
+				.expect(201);
+
+			const responseTaskCard = response.body as TaskCardResponse;
+
+			expect(responseTaskCard.title).toEqual(taskCardParams.title);
+		});
+		it('should throw an error if title is to short', async () => {
+			const user = setupUser([Permission.TASK_CARD_EDIT, Permission.HOMEWORK_CREATE, Permission.HOMEWORK_EDIT]);
+
+			await em.persistAndFlush([user]);
+			em.clear();
+
+			currentUser = mapUserToCurrentUser(user);
+
+			const taskCardParams = {
+				title: 't',
+			};
+
+			await request(app.getHttpServer())
+				.post(`/cards/task/`)
+				.set('Accept', 'application/json')
+				.send(taskCardParams)
+				.expect(400);
+		});
+		// it('should throw an error if title is not a string', async () => {
+		// 	const user = setupUser([Permission.TASK_CARD_EDIT, Permission.HOMEWORK_CREATE, Permission.HOMEWORK_EDIT]);
```

Comment:
@MartinSchuhmacher @virgilchiriac 
There seems to be no effect using the IsString-Decorato in dto/task-card.params.ts Could you check please?

Reply:
interesting indeed.
the `@SanitizeHtml` also only makes the request throw an 'not implemented 501'
not yet any idea to solve that to be honest..

Reply:
maybe @virgilchiriac has another idea

Reply:
probably it does not get to IsString(), if SanitizeHtml(), which is a transformer, gets called first?

As Martin said, SanitizeHtml is also checking for string, and will throw an error if not string. 

But I would keep IsString() check, because SantizeHtml might be extended to support other types in the future.
So, if that's true, test for string only is not needed, but a test to check for sanitization instead makse more sense.
 
https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/apps/server/src/shared/controller/transformer/sanitize-html.transformer.ts#L192

Reply:
I leave it that the IsString-Decorator doesn't work and let the stanize-decorator the work do. But I think it has to be checked.